docs/design/downloads.md

```diff
@@ -0,0 +1,100 @@
+# Edge Image Builder Downloads
+
+## EIB Builder Image Container Downloads
+
+These packages are necessary for building the EIB binary.
+```
+gpgme-devel
+device-mapper-devel
+libbtrfs-devel
```

Comment:
Will this replace the installation information for these packages in the [Prerequisites](https://github.com/dbw7/edge-image-builder/tree/download-design?tab=readme-ov-file#prerequisites) portion of the EIB doc, or is it meant to just describe all the dependencies/downloads that EIB works with?